### ISSUE TITLE
[Bug]: Doesn't work with Fresh Animations

### ISSUE THREAD

**Issue by QwertRvg** [Mrbysco/Heads#4]
### General Info

- [X] I am running a modpack
- [X] I can reproduce this issue consistently in single-player
- [ ] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [X] I am using the latest version of the mod

### Forge version

47.2.0

### Minecraft version

1.20.1

### Mod version

1.1.2

### Java version

CurseForge Default version (i cant find which one that is, sorry)

### Issue Description

So I understand that the mod uses the textures loaded into the game directly, making it Resource Pack compatible.
That does cause a lot of problems when used along side the Fresh Animations resource pack.
Spider Heads are invisible, Horse and Donkey heads are completely invisible and most heads don't have eyes.

Is there no way to circumvent this?

### Additional Information

 
![image](https://github.com/Mrbysco/Heads/assets/89683173/41890120-0df3-4047-9583-6b20a56bfb93)

**Comment by Mrbysco** [comment]
Not really, unless I add direct compatibility (which would be a pain to do for any mod that breaks the mod)

**Comment by Mrbysco** [comment]
I've added a section to the mod description stating it's currently incompatible
